File: typescript/test/Wall/stubWallDAO.ts
import {WallDAOInterface} from '../../src/Wall/WallDAOInterface';
import {User} from '../../src/User/User';
import {Brick} from '../../src/Wall/Brick';

export class StubWallDAO implements WallDAOInterface {
    getBricks(user: User): Brick[] {
        return [];
    }

    addBrick(user: User, brick: Brick): void {
    }
}

File: dotNet/test/Wall/StubWallDAO.cs
using src.Wall;

namespace test.Wall;

public class StubWallDAO : IWallDAO
{
    public List<Brick> GetBricks(src.User.User user)
    {
        return new List<Brick>();
    }

    public void AddBrick(src.User.User user, Brick brick)
    {
    }
}
